src/main/java/com/triviamachine/triviamachine/database/QuestionSchedule.java

```diff
@@ -0,0 +1,65 @@
+package com.triviamachine.triviamachine.database;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.*;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+@Entity
+public class QuestionSchedule {
+    @Id
+    @GeneratedValue
+    private long id;
+
+    @ManyToOne
+    private Question question;
+
+
+    private Date date;
+
+    @OneToOne
+    private Results results;
+
+    public Question getQuestion() {
+        return question;
+    }
+
+    public void setQuestion(Question question) {
+        this.question = question;
+    }
+
+    public Date getDate() {
+        return date;
+    }
+
+    public String getDateToString() {
+        return new SimpleDateFormat("yyyy-MM-dd").format(date);
+    }
+
+    public boolean isAfterToday() throws ParseException {
+        DateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
+        Date today = formatter.parse(formatter.format(new Date()));
```

Comment:
It's best for functions to always return the same results for the same input.  It makes them more testable and predictable.

So, whenever possible, pass in things like "today" dates rather than generating them within your business logic.  "isAfter(Date day)" would have been better.

Also.... why not just this.date.after(new Date())?